Content.Shared/Climbing/Systems/ClimbSystem.cs

```diff
@@ -504,7 +516,25 @@ public void ForciblySetClimbing(EntityUid uid, EntityUid climbable, ClimbingComp
 
     private void OnBuckled(EntityUid uid, ClimbingComponent component, ref BuckledEvent args)
     {
-        StopClimb(uid, component);
+        StopOrCancelClimb(uid, component);
+    }
+
+    private void OnStored(EntityUid uid, ClimbingComponent component, ref ContainerGettingInsertedAttemptEvent args)
+    {
+        StopOrCancelClimb(uid, component);
+    }
+
```

Comment:
Attempt events shouldn't be making state changes, only when the action happens.

Reply:
altered subscription to EntGotInsertedIntoContainerMessage